ci/run.js

```diff
@@ -0,0 +1,51 @@
+var spawn = require("cross-spawn");
+var os = require("os");
+
+var browser = os.platform() === "win32" ? "" : ":phantomjs";
+
+// The pyramid of doom :( If you have a better idea, (which doesn't
```

Comment:
Pyramid of doom haha!
Theoretically, we can even run these in parallel, so when a solution is created to clean up the pyramid, it also has the opportunity of super-improving the test speed

Reply:
Yes, they run quite smoothly in parallel, but the logging gets all screwed up, it gets hard to tell which logging statement belongs to which project...

Reply:
True, though we could do some sort of buffering - [lerna](https://github.com/lerna/lerna) can run commands concurrently but somehow produces useful logs. Either way, not a priority, but a potential improvement :)

Reply:
The pyramid looks a bit like ascii art, doesn't it 🤓